assets/src/edit-story/app/media/utils/useProcessGif.js

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import { useCallback } from 'react';
+import { fetchRemoteFile, isAnimatedGif } from '@web-stories-wp/media';
+
+/**
+ * Internal dependencies
+ */
+import useExistingData from './useExistingData';
+import useUpdateElementDimensions from './useUpdateElementDimensions';
+
+function useProcessGif({
```

Comment:
This hook is very similar to `useProcessVideo`, we should consider merging them into one `useProcessMedia` hook

Reply:
removed